src/cobalt-launcher.c

```diff
@@ -112,23 +112,36 @@ gboolean cobalt_launcher_read_flags_file(CobaltLauncher *launcher, GFile *file,
       continue;
     }
 
-    if (g_str_has_prefix(line, ENABLE_FEATURES_FLAGFILE_PREFIX) ||
-        g_str_has_prefix(line, DISABLE_FEATURES_FLAGFILE_PREFIX)) {
-      const char *feature = strchr(line, '=') + 1;
-      if (*feature == '\0') {
-        g_warning("Line in '%s' has an empty feature: %s", g_file_peek_path(file), line);
+    {
+      gint tokenc;
+      gchar **tokenv;
+      gboolean ret;
+
+      ret = g_shell_parse_argv (line, &tokenc, &tokenv, NULL);
+      if (!ret) {
+        return FALSE;
       }
 
-      cobalt_launcher_set_feature(launcher, feature,
-                                  g_str_has_prefix(line, ENABLE_FEATURES_FLAGFILE_PREFIX)
-                                      ? COBALT_LAUNCHER_FEATURE_ENABLED
-                                      : COBALT_LAUNCHER_FEATURE_DISABLED);
-    } else if (!(g_str_has_prefix(line, FLAG_PREFIX) &&
-                 line[strlen(FLAG_PREFIX)] != '\0')) {
-      g_warning("Line in '%s' is not a flag (must start with '--'): %s",
-                g_file_peek_path(file), line);
-    } else {
-      cobalt_launcher_add_arg(launcher, line);
+      for (int i = 0; i < tokenc; i++) {
+        if (g_str_has_prefix(tokenv[i], ENABLE_FEATURES_FLAGFILE_PREFIX) ||
+            g_str_has_prefix(tokenv[i], DISABLE_FEATURES_FLAGFILE_PREFIX)) {
+          const char *feature = strchr(tokenv[i], '=') + 1;
+          if (*feature == '\0') {
+            g_warning("Argument in '%s' has an empty feature: %s", g_file_peek_path(file), tokenv[i]);
```

Comment:
Any chance you could run this through `clang-format`? I believe some of these lines are now above the line width and need reformatting.